code/modules/butchery/butchery_products_meat.dm

```diff
@@ -0,0 +1,64 @@
+/obj/item/chems/food/butchery/meat
+	name           = "meat"
+	desc           = "A slab of meat."
+	icon           = 'icons/obj/items/butchery/meat1.dmi'
+	icon_state     = "meat"
+	slice_path     = /obj/item/chems/food/rawcutlet
+	slice_num      = 3
+	filling_color  = "#ff1c1c"
+	center_of_mass = @'{"x":16,"y":14}'
+	bitesize       = 3
+	utensil_flags  = UTENSIL_FLAG_COLLECT | UTENSIL_FLAG_SLICE
+	drying_wetness = 60
+	dried_type     = /obj/item/chems/food/jerky/meat
+	nutriment_type = /decl/material/solid/organic/meat
```

Comment:
this feels dangerous, `nutriment_type` expects a `/decl/material/liquid/nutriment` path and will call procs expecting that